oci/caps/utils.go

```diff
@@ -8,14 +8,20 @@ import (
 	"github.com/syndtr/gocapability/capability"
 )
 
-var capabilityList Capabilities
+var (
+	allCaps []string
+
+	// capabilityList maps linux capability name to its value of capability.Cap
+	// type. This list contains nil entries for capabilities that are known, but
+	// not supported by the current kernel.
+	// Capabilities is one of the security systems in Linux Security Module (LSM)
+	// framework provided by the kernel.
+	// For more details on capabilities, see http://man7.org/linux/man-pages/man7/capabilities.7.html
+	capabilityList map[string]*capability.Cap
```

Comment:
I think we should now use `oci.WithAllCurrentCapabilities`
https://github.com/containerd/containerd/blob/01a39262e8c6f8ac865722a8b8343bd85b24452c/oci/spec_opts_linux.go#L126-L140

Reply:
With https://github.com/opencontainers/runc/pull/2854, I think we should be able to remove checks altogether (once released), correct?

Reply:
Yes :+1:

(But it should be probably noted that some distro packagess may provide newer dockerd with older runc)